core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -455,6 +454,14 @@ sealed abstract private[data] class KleisliInstances8 extends KleisliInstances9
 sealed abstract private[data] class KleisliInstances9 {
   implicit def catsDataFunctorForKleisli[F[_], A](implicit F0: Functor[F]): Functor[Kleisli[F, A, ?]] =
     new KleisliFunctor[F, A] { def F: Functor[F] = F0 }
+
+  implicit def catsDataCoflatMapForKleisli[F[_], In](implicit F0: Applicative[F]): CoflatMap[Kleisli[F, In, ?]] =
```

Comment:
This one has to go one level above `Functor` since `Coflatmap` inherits `Functor` and thus is more "specific" type wise. Let's move it to ` KleisliInstances8` .

Reply:
Cheers - that's been done